src/components/FlyoutMenu/FlyoutMenu.test.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { fireEvent, render, waitFor, waitForElement } from '@testing-library/react';
+
+import FlyoutMenu, { FlyoutMenuDirection } from './FlyoutMenu';
+
+const applyButtonText = 'Apply';
+const cancelButtonText = 'Cancel';
+
+describe('FlyoutMenu', () => {
+  it('Renders an open transactional menu', async () => {
```

Comment:
Why are the tests using `async` and   `await waitForElement`? Is it really needed?